src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -65,9 +65,6 @@ public void beforeAll() throws Exception {
 
     stageWithPatternName = "ingest_sdk_test_stage_pattern" + num;
 
-    TestUtils.executeQuery("use database SNOWPIPE_SDK_DB");
```

Comment:
Since we are passing the database and schema in the profile.json, not sure if we need to hard code it here.